lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -409,6 +411,44 @@ withBFTNode tr baseDir (NodeParams severity systemStart (port, peers)) action =
     name = "bft"
     dir = baseDir </> name
 
+withPassiveNode
```

Comment:
It could be nice to add a comment explaining _why_ we are using this indirection.